src/components/App/App.js

```diff
@@ -62,7 +62,25 @@ export default class App extends Component {
           logOut={this.logOut}
           movies={this.state.movies}
           addMovies={this.addMovies}/>} />
-          
+
```

Comment:
We need to have addFavorite function such this: 
```
addFavorite = (movie) => {
    this.setState({favorite: [...this.state.favorite, movie]})
}

```
And pass it to child components until favorite button

Reply:
I was just doing Iteration 3 for this push, we can add iteration 4 today